cwltool/main.py

```diff
@@ -63,7 +65,8 @@ def arg_parser():  # type: () -> argparse.ArgumentParser
     parser.add_argument("--no-container", action="store_false", default=True,
                         help="Do not execute jobs in a Docker container, even when specified by the CommandLineTool",
                         dest="use_container")
-
+    parser.add_argument("--parallel", action="store_true", default=False,
+                        help="Run jobs in parallel")
```

Comment:
"Experimental: Run jobs in parallel. Does not currently take ResourceRequirments into account."

Reply:
Should we mark it as experimental? Since we warn that it does not take ResourceRequirements into account, is there any other behavior one would expect from this option that it doesn't provide?

Reply:
Since it could leads to dozens or hundreds of simultaneously tool executions on someone's laptop, then it really should be labeled as being experimental or dangerous

Maybe something more explicit like

> Experimental: Run jobs in parallel. Does not currently keep track of ResourceRequirments like the number of cores or memory and could overload this system.

would be better